src/main/backend.ts

```diff
@@ -237,11 +294,13 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
       contents: Electron.WebContents,
       details?: Electron.RenderProcessGoneDetails,
     ): Promise<void> => {
-      try {
-        await this._sendNativeCrashes(this._getNewEventWithElectronContext(contents, details));
-      } catch (e) {
-        // eslint-disable-next-line no-console
-        console.error(e);
+      if (this._options.useSentryMinidumpUploader !== false) {
```

Comment:
This is purely for my education: How does this flag relate to `useCrashpadMinidumpUploader`?

Reply:
I'm not entirely sure since I didn't add them!

My best guess is that it's so you can have minidump capture enabled but disable uploading with both set to false.